content/zh/docs/reference/_index.md

```diff
@@ -48,8 +59,10 @@ client libraries:
 - [Kubernetes Python client library](https://github.com/kubernetes-client/python)
 - [Kubernetes Java client library](https://github.com/kubernetes-client/java)
 - [Kubernetes JavaScript client library](https://github.com/kubernetes-client/javascript)
+- [Kubernetes Dotnet client library](https://github.com/kubernetes-client/csharp)
```

Comment:
Csharp is not Dotnet.

https://docs.microsoft.com/en-us/dotnet/core/introduction#programming-languages

Reply:
>The generated code works on all platforms supported by .NET or .NET Core.

https://github.com/kubernetes-client/csharp#generating-the-client-code

Reply:
https://github.com/kubernetes/website/pull/28250

😅The java client compiles code for all JVM platform languages, 
but it's still called the java client, not the JVM client.